packages/devtools/lib/src/inspector/inspector_service.dart

```diff
@@ -241,7 +221,7 @@ class InspectorService {
   }
 
   void onExtensionVmServiceRecieved(Event e) {
-    if ('Flutter.Frame' == e.kind) {
+    if ('Flutter.Frame' == e.extensionKind) {
```

Comment:
flat out bug. we didn't need to listen for the flutter frame because it happened that the app would always start after the first frame was already rendered so this code path didn't matter.

Reply:
Are both `kind` and `extensionKind` strings? If no, we should enable the lint that only allows comparisons between same types.

Reply:
Unfortunately they are both strings but it looks like at one point we wanted to make kind be an enum type.
```dart
/*EventKind*/ String kind;
```
We do have `test_types_in_equals` enabled. I'm happy to know this lint exists as that is often something I worry about when writing an `==`